lesson6/public/index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title></title>
+  </head>
+  <body>
+    Name:   <input type = 'text' id = 'inputBox'/>
+    Comment: <input type = 'text' id = 'comBox'/>
+    <input type = 'submit' id = 'subBox'/>
+    <div>
+      <pre id = 'm1'/>
+ </div> 
+ <script>
+const button = document.getElementById('subBox');
+button.onclick=()=>{
+  const name = document.getElementById('inputBox').value;
+  const comment = document.getElementById('comBox').value;
+  const xhttp = new XMLHttpRequest();
+  xhttp.open('GET', `/XMLHttpRequest?nameVal=${name}&comVal=${comment}`);
```

Comment:
/XMLHttpRequest can be a bit simpler , so that you wont get confused.

Reply:
Thanks Madhu! will change the name of the request